cg/meta/qc_metrics/collect_qc_metrics.py

```diff
@@ -0,0 +1,85 @@
+"""Module for the collect qc metrics meta api."""
+
+import logging
+from typing import Iterator
+
+from housekeeper.store.models import File, Tag
+
+from cg.clients.arnold.api import ArnoldAPIClient
+from cg.clients.janus.api import JanusAPIClient
+from cg.clients.janus.dto.create_qc_metrics_request import (
+    CreateQCMetricsRequest,
+    FilePathAndTag,
+)
+from cg.clients.janus.exceptions import JanusClientError, JanusServerError
+from cg.exc import HousekeeperFileMissingError
+from cg.meta.meta import MetaAPI
+from cg.models.cg_config import CGConfig
+from cg.store.models import Case, Sample
+
+LOG = logging.getLogger(__name__)
+
+
+class CollectQCMetricsAPI(MetaAPI):
+    """Collect qc metrics API."""
+
+    def __init__(self, config: CGConfig):
+        super().__init__(config)
+        self.janus_api: JanusAPIClient = config.janus_api
+        self.arnold_api: ArnoldAPIClient = config.arnold_api
+
+    def get_qc_metrics_file_paths_for_case(self, case_id: str) -> list[File]:
```

Comment:
Should this return the paths or the whole File objects? Method name seems to suggest only paths.

Reply:
fixed